README.md

```diff
@@ -19,28 +24,53 @@ This is an **experimental** fork of [Sebastián Ramírez's](https://github.com/t
 - [Help needed](#help-needed)
 - [Release notes](#release-notes)
 - [License](#license)
-  
-## Screenshots
 
-### App landing page
 
-![Landing page](img/landing.png)
+## Requirements
+
+Please make sure you have these installed before proceeding!
+* [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+* [Python 3.11+](https://www.python.org/downloads/)
+* [Cookiecutter](https://cookiecutter.readthedocs.io/en/stable/) -- The Python Cookiecutter library.
+* [Hatch](https://hatch.pypa.io/latest/)
+* [Node.js](https://nodejs.org/en)
+* [MongoDB Compass](https://www.mongodb.com/products/tools/compass) -- To help visualize local/remote database easily.
+
+## QuickStart
 
-### Dashboard Login
+For those that what to dive in and play around with the generated code, here's quick start guide on how to do it. It is advised, though, once you have run through generating this app the first time, that you circle back and check out to more fleshed out [Building a Generated App](./docs/getting-started.md) section. **NOTE: this will generate a local application that is not yet production-ready. Please go through all of the README information linked before creating a version to deploy to a production host**
 
-![Magic-link login](img/login.png)
+```
+// Make sure cookiecutter and python are installed on the device
+// This will generate a full-stack app in the directory ./example
+cookiecutter https://github.com/mongodb-labs/full-stack-fastapi-mongodb --no-input project_name="example"
 
-### Dashboard User Management
+// change into that directory
+cd example
 
-![Moderator user management](img/dashboard.png)
+// Make sure that you've opened Docker before this step
+// build
+docker compose build --no-cache
 
-### Interactive API documentation
+// start the container in the background
+docker compose up -d 
+```
 
-![Interactive API docs](img/redoc.png)
+Now, you can view the site by going to `localhost:3000`. You can also see all the logs of the running containers in your Docker Desktop app.
 
-### Enabling two-factor security (TOTP)
+The will only be one user on the site, `admin@example.com`. If you choose a different project name, then the email domain changes. i.e. `project_name=fullstackexample` the user email will be `admin@fullstackexample.com`. The default password will be set to `changethis` but that can be changed. 
```

Comment:
"The will" -> "There will"

Reply:
Use a colon instead of a comma after "site"

Reply:
"i.e. ..." -> "For example, if you create your project using ..."

Reply:
We should call out that users should change their default password immediately, even for local testing.

Reply:
> We should call out that users should change their default password immediately, even for local testing.

Ah yeah, this is the quickstart guide, so I felt it unnecessary here. In the later documents it talks about making sure to set up a good hash and how to do so.